app/controllers/waste_carriers_engine/govpay_forms_controller.rb

```diff
@@ -55,6 +55,7 @@ def respond_to_acceptable_payment(action)
         redirect_to_correct_form
       else
         log_and_send_govpay_response(false, action)
+        action = GovpayPaymentDetailsService.payment_status(action)
```

Comment:
Does this line do anything?

Reply:
there's a class method in `GovpayPaymentDetailsService` that maps some actions which match up with the locales currently defined.

```ruby
def self.payment_status(status)
  {
    "created" => :pending,
    "started" => :pending,
    "submitted" => :pending,
    "cancelled" => :cancel,
    "failed" => :failure,
    nil => :error
  }.freeze[status] || status.to_sym
end
```